JSTests/stress/intl-displaynames-v2.js

```diff
@@ -47,7 +47,7 @@ function shouldThrow(func, errorMessage) {
     shouldBe(dn.of("gregory"), "Gregorian Calendar");
     shouldBe(dn.of("ethioaa"), "Ethiopic Amete Alem Calendar");
     shouldBe(dn.of("japanese"), "Japanese Calendar");
-    shouldBe(dn.of("dangi"), "Dangi Calendar");
+    shouldBe(dn.of("dangi"), "Korean Calendar");
```

Comment:
For newer data, please accept both since we need to support some builds with older ICU.